extensions/prost/private/prost.bzl

```diff
@@ -233,7 +234,10 @@ def _rust_prost_aspect_impl(target, ctx):
 
     rustfmt_toolchain = ctx.toolchains["@rules_rust//rust/rustfmt:toolchain_type"]
     prost_toolchain = ctx.toolchains[TOOLCHAIN_TYPE]
-    for prost_runtime in [prost_toolchain.prost_runtime, prost_toolchain.tonic_runtime]:
+    runtimes = [prost_toolchain.prost_runtime, prost_toolchain.tonic_runtime]
+    if not prost_toolchain.compile_well_known_types:
+        runtimes.append(prost_toolchain.prost_types)
```

Comment:
Why explicitly add `prost_types` in this case?

Reply:
The code-generated library will now depend on `prost-types`, so I'm adding the dependency here. Since the user will already have provided a `prost-types` dependency as an argument to the toolchain, I felt it more consistent to automatically add it to the runtimes for them, instead of asking them to provide it again in the runtimes themselves.